docs/autofish.rst

```diff
@@ -0,0 +1,42 @@
+autofish
+========
+
+.. dfhack_tool::
+    :summary: Keep a specific quantity of fish in your fortress.
+    :tags: fort auto fps jobs
```

Comment:
```suggestion
    :tags: fort auto fps labors
```

Reply:
right? it's affecting labor assignment, not the actual job

Reply:
That's right.